pages/home/auth-providers/oauth2.mdx

```diff
@@ -28,19 +28,27 @@ The only supported OAuth 2.0 flow is the authorization code grant flow (with or
 
 ## Configuring OAuth 2.0
 
-How you configure the OAuth 2.0 provider depends on whether you use the Arcade Cloud Engine or a [self-hosted Engine](/home/install/overview).
+How you configure the OAuth 2.0 provider depends on whether you use the Arcade Cloud Engine or a [self-hosted Engine](/home/install/overview). If you use the Cloud Engine, you must configure your provider in the Dashboard.
 
-### Configuring OAuth 2.0 with the Arcade Dashboard
+<Note>
+When configuring your app in the OAuth 2.0 enabled service, you must use `https://cloud.arcade.dev/api/v1/oauth/callback` as the redirect URL (sometimes called callback URL).
+</Note>
 
-1. Navigate to the OAuth section of the Arcade Dashboard and click **Add OAuth Provider**.
+### Using the Arcade Dashboard
+
+1. Navigate to the OAuth section of the [Arcade Dashboard](https://api.arcade.dev/dashboard/home) and click **Add OAuth Provider**.
```

Comment:
we need to have a different url for self-hosted or just have a link for the cloud.

Reply:
this is under `### Arcade Cloud Engine`, so I guess we should only have the cloud URL here?

Reply:
What would you suggest, @wdawson? I'm confused about which heading and what to put in the instructions

Reply:
I might drop the link entirely (like it was before). Or something like
```suggestion
1. Navigate to the OAuth section of the Arcade Dashboard ([here](https://api.arcade.dev/dashboard/home) if using Arcade cloud) and click **Add OAuth Provider**.
```

Over time, maybe we need a docs page about how to access the dashboard during a self-hosted installation